### ISSUE TITLE
ListSelect does not work with keyboard navigation

### ISSUE THREAD

**Issue by Artur-** [vaadin/framework8-issues#463]
1. Click on a list select
2. Press arrow down

Expected: value is updated on the server
Actual: selection is changed in the browser, nothing is sent to the server
